templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -13,7 +13,9 @@
    "cell_type": "markdown",
```

Comment:
<p>This link doesnt work</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/86/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Ah good catch, because we moved this over from <code>ark</code> our docs site no longer supports this. I'm not sure if we have plans to set up an equivalent docs site for this repo, so I'll just copy the documentation we have internally for this procedure here.</p>

Reply:
@HPiyadasa on second thought, I'm just gonna delete this. Users are likely not going to be needing Moly point separators anymore, and the description is straightforward enough to not need a separate link with additional info not contained in the existing documentation for this cell.